### PR TITLE
Weatherradio v 0.8 | Documentation added

### PR DESCRIPTION
**Pull request by sterne-jaeger**
Here comes the first version of the documentation - see doc/Readme-WeatherRadio.md. Additionally, the firmware configuration has been streamlined so that all configuration changes can be done from config.h.

### CODE REVIEW

File: indi-duino/add-on/weatherradio/bin/wr_config.py [hunk 1]
```diff
@@ -24,13 +24,13 @@
 INDIDEVICEPORT="/dev/ttyUSB0"
 
 # RRD storage holding time series data
-RRDFILE="./weather.rrd"
+RRDFILE="/usr/local/share/weatherradio/weather.rrd"
```
Comment (knro) [inline indi-duino/add-on/weatherradio/bin/wr_config.py]: is there a reason this is /usr/local/share and not /usr/share? The reason is if we make a debian package for this, it should be install to /usr/share

Comment (sterne-jaeger) [reply indi-duino/add-on/weatherradio/bin/wr_config.py]: Fine for me. I have my stuff always under /usr/local/... I will correct it.

There is also a bugfix for the firmware, so please wait with merging.

**Comment by sterne-jaeger** [comment]
OK, path changed in the config files and in the documentation.

**Comment by knro** [comment]
Thanks, is this ready to land now?

**Comment by sterne-jaeger** [comment]
Yes, everything is ready from my side.

**Pull request merged**